test/test_metalearning/pyMetaLearn/test_meta_features_sparse.py

```diff
@@ -114,9 +109,18 @@ def sparse_data_transformed():
     X_transformed = standard_scaler.fit_transform(X_transformed)
 
     # Transform the array which indicates the categorical metafeatures
-    number_numerical = np.sum(~np.array(list(categorical.values())))
-    categorical_transformed = {
-        i: True if i < (X_transformed.shape[1] - number_numerical) else False
+    number_not_categorical = np.sum(
+        np.array(
+            [
+                False if value.lower() == "categorical" else True
+                for value in feat_type.values()
+            ]
+        )
+    )
+    feat_type_transformed = {
```

Comment:
i was also confused about this one. But i just replace "True" with "categorical" and "False" with "numerical". So the behavoir of categorical_tranforemd aka. feat_type transformed is somehow weird. And i think the test fail if i change this. But this can also be because the test are testing for the wrong values.

Reply:
The test appears to be wrong. After transformation all features should be numerical, and there should be no categorical features left.